tests/testthat/test-git.R

```diff
@@ -0,0 +1,23 @@
+test_that("test that calling git_checkout with correct branch name and an exist local path of github project returns list of string", {
```

Comment:
Hi Malia,

I am not sure if I am missing something here, but some of the past revision requests are still pending on your tests. For example, I still see the tests are "test _that (test that)". I recall mentioning this before.